src/sql/parts/notebook/notebookActions.ts

```diff
@@ -227,6 +227,25 @@ export class TrustedAction extends ToggleableAction {
 	}
 }
 
+// Action to run all code cells in a notebook.
+export class RunAllCellsAction extends Action {
+	constructor(
+		id: string, label: string, cssClass: string
+	) {
+		super(id, label, cssClass);
+	}
+	public run(context: NotebookComponent): Promise<boolean> {
```

Comment:
nit: Making this async is a lot less verbose

Reply:
I hear ya. But this is a problem with Actions all up, right?